src/scenedit/scen.main.cpp

```diff
@@ -84,10 +92,90 @@ extern void set_up_apple_events();
 std::unordered_map<std::string, std::shared_ptr <iEventListener>> event_listeners;
 cDrawableManager drawable_mgr;
 
+fs::path scenedit_dir;
+fs::path game_dir;
+fs::path game_binary;
+extern std::string last_load_file;
+
+enum eLaunchType {LOC,START,ENTRANCE};
```

Comment:
Please either make this an `enum class` or add a prefix to each name, eg something like `LAUNCH_LOC` etc.